src/main/resources/db/migration/all/20220803121541__characteristic_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE characteristic (
```

Comment:
Just for consistency sake with the other tables, could we name this `characteristics` instead?

Reply:
I think I could have over communicated [the naming choices more Ian](https://mojdt.slack.com/archives/D042D4ME75F/p1666619714158769). We learnt about the pluralised snake cased naming and [documented all the models here](https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/main/doc/architecture/decisions/0006-rooms-and-beds-for-both-cas-services.md#decision).